src/__tests__/__snapshots__/wait-for-element-to-be-removed.js.snap

```diff
@@ -0,0 +1,51 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I don't think these snapshots are at all necessary and just make it harder to know the intent of the tests. Could we change these to be regular assertions instead?

Reply:
@kentcdodds I have almost removed the snapshot to inline snapshots(Which should be working as regular assertions). But, because we are also passing the error we got from the `getByTestId` etc, in that case, a snapshot will be a good thing I guess.